### DISCUSSION TITLE
Known issue with transient prompt, tooltips and zsh 5.9?

### DISCUSSION THREAD

**Discussion by DeepSpace2** [JanDeDobbeleer/oh-my-posh#5071] [Troubleshoot] [answered]
Hi, before opening an full-blown issue I wanted to discuss if it is a known issue for transient prompt and tooltips to not work with zsh 5.9. I understood this version of zsh is a bit problematic but the docs do not specifically point this out.

Thanks.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@DeepSpace2 this is something on your end as this works for me on 5.9. Can you have a look at if the `oh-my-posh init` command is added as the last statement in your `.zshrc`? Otherwise other tools can override our key handlers.

**Reply by DeepSpace2** [reply]
🤦 it was actually the very first line :D I just added it there and thought it is fine since everything else worked. Thanks
